test/overlay.html

```diff
@@ -411,54 +424,55 @@
     describe('revealDate', () => {
       let overlay, monthScroller;
 
-      function fixtureOverlayContent({monthScrollerItems, monthScrollerOffset}, done) {
-        overlay = fixture('overlay');
-        monthScroller = overlay.$.monthScroller;
-        monthScroller.style.setProperty('--vaadin-infinite-scroller-buffer-offset', monthScrollerOffset);
-        monthScroller.updateStyles({'--vaadin-infinite-scroller-buffer-offset': monthScrollerOffset});
-        monthScroller.style.height = `${270 * monthScrollerItems}px`;
-        overlay.i18n = getDefaultI18n();
-        overlay.$.monthScroller.bufferSize = 3;
-        overlay.$.yearScroller.bufferSize = 3;
-        overlay.initialPosition = new Date(2021, 1, 1);
-        setTimeout(() => {
-          Polymer.RenderStatus.afterNextRender(overlay, done);
-        }, 1);
-      }
-
       describe('height(visible area) < height(item)', () => {
         beforeEach((done) => {
-          fixtureOverlayContent({
+          overlay = customizeFixture({
+            initialPosition: new Date(2021, 1, 1),
             monthScrollerItems: 0.5,
             monthScrollerOffset: 0
-          }, done);
-        });
-
-        it('should scroll when the month is above the visible area', () => {
-          const position = monthScroller.position;
+          });
+          monthScroller = overlay.$.monthScroller;
+          setTimeout(() => {
+            Polymer.RenderStatus.afterNextRender(overlay, done);
+          }, 1);
+        });
+
+        it('should scroll to a position that approximately shows the week the date is in', () => {
+          // Starting on first of February
+          const initialPosition = monthScroller.position;
+          // Scroll to 15th
+          overlay.revealDate(new Date(2021, 1, 15), false);
+          const positionOf15th = monthScroller.position;
+          expect(positionOf15th).to.be.greaterThan(initialPosition);
+          expect(positionOf15th).to.be.lessThan(initialPosition + 1);
```

Comment:
Spent some time to create a test that checks if the date element is actually visible in the scroll area, but could not find a reliable setup. The scroller hard-codes the height of an item to 270px, while the test does not load the theme, which leads to an item only using about 135px.

So I went with the same position property checks that the other tests use.